Formula/gdal-20.rb

```diff
@@ -70,6 +71,7 @@ class Gdal20 < Formula
   end
 
   depends_on :java => ["1.7+", :optional, :build]
+  depends_on "numpy" => :python||:python3
```

Comment:
Does this conditional work? I could not find another instance of it in any other formula.